clickhouse-admin/src/bin/clickhouse-admin-single.rs

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Single-node ClickHouse database admin binary.
+
+use anyhow::anyhow;
+use camino::Utf8PathBuf;
+use clap::Parser;
+use omicron_clickhouse_admin::{ClickhouseCli, Config};
+use omicron_common::cmd::fatal;
+use omicron_common::cmd::CmdError;
+use std::net::{SocketAddr, SocketAddrV6};
+
+#[derive(Debug, Parser)]
+#[clap(
+    name = "clickhouse-admin-single",
+    about = "Single-node ClickHouse admin server"
+)]
+enum Args {
+    /// Start the single-node ClickHouse admin server
+    Run {
+        /// Address on which this server should run
+        #[clap(long, short = 'a', action)]
+        http_address: SocketAddrV6,
+
+        /// Path to the server configuration file
+        #[clap(long, short, action)]
+        config: Utf8PathBuf,
```

Comment:
I think you can remove this flag. We won't be generating the config file with clickward

Reply:
aha! you're right nvm